references/cli/top/component/table.go

```diff
@@ -56,7 +56,6 @@ func (t *Table) Start() {
 
 // Stop table component
 func (t *Table) Stop() {
-	t.Clear()
```

Comment:
why removed

Reply:
First, the logic of rendering table is that the rendering of table header and table body are separate, because the table header is same of the same kind of resource table, so header will be rendered only once.

I think i can clear the body when table is stopped, so i clear it. But i find table header is cleared in the same time.  In the next time rendering, the header will disappear. So I decide to remove the `Clear`.


But just now, I find i also need `Clear`, because if i don't clear the table body, in the case  that the older table is longer  and the latter table is short, the older table's data will be left in new table.  

Thanks for helping me fix a bug! 😆

Reply:
can we add a test to prevent it happen next time especially when other maintainer contribute this part?

Reply:
A test about logic here? 

For example, the content of table is empty when it is stopped?

Reply:
Maybe your idea is add more test case to prevent bugs happened? Ok, i will  add more test cases as i can.